shuup/simple_cms/models.py

```diff
@@ -37,9 +38,13 @@ def visible(self, dt=None):
         qs = self.filter(q)
         return qs
 
+    def for_shop(self, shop):
```

Comment:
Since we don't offer shared pages should we just pass the shop to visible queryset filter as well. Now I need to know to separately (basically always) also call this for shop queryset filter.